src/ppm.cpp

```diff
@@ -245,19 +245,19 @@ void PPM::print_configuration() const
 bool PPM::configure() {
 
     if ( optIsSet('v') ) {
-        if ( "trace" == optString('v') ) {
+        if ( "TRACE" == optString('v') ) {
```

Comment:
Can we make this log level string case insensitive? I see the configuration doc mentioned -v | --log-level : The info log level [trace,debug,info,warning,error,critical,off] They are all lower case. The code only compare the upper case.